pkg/validation/metadata.go

```diff
@@ -0,0 +1,43 @@
+package validation
+
+import (
+	"fmt"
+	"github.com/devfile/api/v2/pkg/devfile"
+	"strings"
+)
+
+var validArchitectures = map[string]bool{
```

Comment:
+1

Good to keep the list of supported arch's small for now. We're very unlikely to expand beyond these 4 (even arm64 is unlikely I'd say).